### PR TITLE
Dup - TotallyUsefulSoftware,Inc.QuickSFV version 3.0.0

### PR DESCRIPTION
**Pull request by Masamune3210**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

### CODE REVIEW

File: manifests/t/TotallyUsefulSoftware,Inc/QuickSFV/3.0.0/TotallyUsefulSoftware,Inc.QuickSFV.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+PackageIdentifier: TotallyUsefulSoftware,Inc.QuickSFV
+PackageVersion: 3.0.0
+Installers:
+  InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: msi
+  InstallerUrl: https://www.quicksfv.org/quicksfv-setup32.msi
+  InstallerSha256: 303AEDA03E64121EA91E53F9C541CB5B5C0E6BA104E1CC8FD8FA4CB7F3E3D704
+- InstallerLocale: en-US
+  Architecture: x64
```
Comment (vedantmgoyal2009) [inline manifests/t/TotallyUsefulSoftware,Inc/QuickSFV/3.0.0/TotallyUsefulSoftware,Inc.QuickSFV.installer.yaml:13]:
```suggestion
Installers:
  InstallerLocale: en-US
  Architecture: x86
  InstallerType: msi
  InstallerUrl: https://www.quicksfv.org/quicksfv-setup32.msi
  InstallerSha256: 303AEDA03E64121EA91E53F9C541CB5B5C0E6BA104E1CC8FD8FA4CB7F3E3D704
  ProductCode: '{DBDCD3AF-20E4-4E5E-80E8-B14109FE5DD9}'
- InstallerLocale: en-US
  Architecture: x64

```

**Review by vedantmgoyal9 — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
https://github.com/microsoft/winget-pkgs/pull/27046#event-5258238394

File: manifests/t/TotallyUsefulSoftware,Inc/QuickSFV/3.0.0/TotallyUsefulSoftware,Inc.QuickSFV.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: msi
+  InstallerUrl: https://www.quicksfv.org/quicksfv-setup32.msi
+  InstallerSha256: 303AEDA03E64121EA91E53F9C541CB5B5C0E6BA104E1CC8FD8FA4CB7F3E3D704
+  ProductCode: '{DBDCD3AF-20E4-4E5E-80E8-B14109FE5DD9}'
+- InstallerLocale: en-US
+  Architecture: x64
+
+  InstallerType: msi
```
Comment (vedantmgoyal2009) [inline manifests/t/TotallyUsefulSoftware,Inc/QuickSFV/3.0.0/TotallyUsefulSoftware,Inc.QuickSFV.installer.yaml:16]:
```suggestion
  Architecture: x64
  InstallerType: msi
```

**Review by vedantmgoyal9 — CHANGES_REQUESTED** [review]
I'm really sorry, my suggestion contained a new line. Sorry.

File: manifests/t/TotallyUsefulSoftware,Inc/QuickSFV/3.0.0/TotallyUsefulSoftware,Inc.QuickSFV.installer.yaml (hunk 2 above)
Comment (Masamune3210) [reply manifests/t/TotallyUsefulSoftware,Inc/QuickSFV/3.0.0/TotallyUsefulSoftware,Inc.QuickSFV.installer.yaml:16]: Your fine, I didnt notice it either

File: manifests/t/TotallyUsefulSoftware,Inc/QuickSFV/3.0.0/TotallyUsefulSoftware,Inc.QuickSFV.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: TotallyUsefulSoftware,Inc.QuickSFV
+PackageVersion: 3.0.0
+Installers:
+  InstallerLocale: en-US
```
Comment (ItzLevvie) [inline manifests/t/TotallyUsefulSoftware,Inc/QuickSFV/3.0.0/TotallyUsefulSoftware,Inc.QuickSFV.installer.yaml:7]:
```suggestion
- InstallerLocale: en-US
```

File: manifests/t/TotallyUsefulSoftware,Inc/QuickSFV/3.0.0/TotallyUsefulSoftware,Inc.QuickSFV.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: TotallyUsefulSoftware,Inc.QuickSFV
+PackageVersion: 3.0.0
+PackageLocale: en-US
+Publisher: Totally Useful Software, Inc.
+PackageName: QuickSFV
+License: Unknown
+ShortDescription: QuickSFV
```
Comment (ItzLevvie) [inline manifests/t/TotallyUsefulSoftware,Inc/QuickSFV/3.0.0/TotallyUsefulSoftware,Inc.QuickSFV.locale.en-US.yaml:10]:
```
ShortDescription: QuickSFV
```
Please write a better description for what this application is about. You can grab these from the publisher's website.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/t/TotallyUsefulSoftware,Inc/QuickSFV/3.0.0/TotallyUsefulSoftware,Inc.QuickSFV.locale.en-US.yaml (hunk 4 above)
Comment (Masamune3210) [reply manifests/t/TotallyUsefulSoftware,Inc/QuickSFV/3.0.0/TotallyUsefulSoftware,Inc.QuickSFV.locale.en-US.yaml:10]: Thats what im going to do from now on, didn't think about doing that until you mentioned it

File: manifests/t/TotallyUsefulSoftware,Inc/QuickSFV/3.0.0/TotallyUsefulSoftware,Inc.QuickSFV.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: TotallyUsefulSoftware,Inc.QuickSFV
```
Comment (ItzLevvie) [inline manifests/t/TotallyUsefulSoftware,Inc/QuickSFV/3.0.0/TotallyUsefulSoftware,Inc.QuickSFV.installer.yaml:4]:
```suggestion
PackageIdentifier: TotallyUsefulSoftware.QuickSFV
```

File: manifests/t/TotallyUsefulSoftware,Inc/QuickSFV/3.0.0/TotallyUsefulSoftware,Inc.QuickSFV.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: TotallyUsefulSoftware,Inc.QuickSFV
```
Comment (ItzLevvie) [inline manifests/t/TotallyUsefulSoftware,Inc/QuickSFV/3.0.0/TotallyUsefulSoftware,Inc.QuickSFV.locale.en-US.yaml:4]:
```suggestion
PackageIdentifier: TotallyUsefulSoftware.QuickSFV
```

File: manifests/t/TotallyUsefulSoftware,Inc/QuickSFV/3.0.0/TotallyUsefulSoftware,Inc.QuickSFV.yaml [hunk 7]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.3.0.6
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: TotallyUsefulSoftware,Inc.QuickSFV
```
Comment (ItzLevvie) [inline manifests/t/TotallyUsefulSoftware,Inc/QuickSFV/3.0.0/TotallyUsefulSoftware,Inc.QuickSFV.yaml:4]:
```suggestion
PackageIdentifier: TotallyUsefulSoftware.QuickSFV
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Oops, I forgot about this as well.

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Make sure to rename the files as well.

Alternatively, if you're finding that difficult, you can create another PR using wingetcreate.

**Comment by Masamune3210** [comment]
I'll honestly probably just make a new pr, at least until I can have some time to wrap my head around git and how to do things a bit more

**Pull request closed without merging**